src/merkle/merkle-car-factory.ts

```diff
@@ -2,22 +2,32 @@ import type { DiagnosticsLogger } from '@ceramicnetwork/common'
 import { CARFactory, CAR } from 'cartonne'
 import * as DAG_JOSE from 'dag-jose'
 import {
-  IpfsLeafCompare,
   MerkleTreeFactory,
   IpfsMerge,
   type MerkleTree,
   type Node,
   type PathDirection,
   type CIDHolder,
-  type TreeMetadata,
+  type TreeMetadata, CompareFunction,
 } from '@ceramicnetwork/anchor-utils'
-import { BloomMetadata } from './bloom-metadata.js'
 import type { Candidate } from '../services/candidate.js'
 import type { CID } from 'multiformats/cid'
+import {ICandidate} from "@ceramicnetwork/anchor-utils/lib/merkle/candidate.type.js";
```

Comment:
That probably will break in production.

Reply:
Huh, I am positively surprised the code passed CI.

Ah, it is `type`! That's why!

Reply:
ooh good catch, fixed.